src/utils/getURL.ts

```diff
@@ -0,0 +1,4 @@
+export const getURL = () => {
```

Comment:
can we use this for the metadata fields too? e.g. anywhere that `https://onchainsummer.xyz` is hard coded

Reply:
Good point, resolved.